bridge_adaptivity/module/policies/policy_points_earned.py

```diff
@@ -0,0 +1,12 @@
+from .base import BaseGradingPolicy
+
+
+class PointsEarnedGradingPolicy(BaseGradingPolicy):
+    """GRadding policy class calculate grade based upon users earned points."""
```

Comment:
typo s/GRadding/Gradding/

Reply:
actually there should be only one letter ```D``` in word ```grading```